index.html

```diff
@@ -181,6 +181,31 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+  var fireballSize = 22;
+  var getFireballSpeed = function(left) {
+    if (left) {
+      return 5;
+    } else {
+        return 2;
+      };
+
+  };
+  var wizardSpeed = 3;
+  var wizardWidth = 70;
+  var getWizardHeight = function(width) {
+     return(width * 1.337);
+  };
+  var getWizardX = function(width) {
+    return(width/2);
+  };
+  var getWizardY = function(height) {
+    return(height*0.33);
```

Comment:
```suggestion
    return(height - getWizardHeight()) * (2 / 3);
```
нужно учитывать и высоту мага

Reply:
Исправил